packages/ui-extensions/src/surfaces/checkout/components/Chat/Chat.doc.ts

```diff
@@ -5,7 +5,7 @@ const data: ReferenceEntityTemplateSchema = {
   description: 'Use Chat to create live chat applications.',
   thumbnail: 'chat-thumbnail.png',
   requires:
-    'configuration of the [preloads](/docs/api/checkout-ui-extensions/unstable/configuration#preloads-definition) value in the extensions configuration file.',
+    'access to the **Chat in checkout extensions** scope. Request access in the Partner Dashboard.',
```

Comment:
I could not do what you had suggested. The UI requires to start with "Requires ...".